packages/@react-spectrum/s2/src/Avatar.tsx

```diff
@@ -22,20 +22,33 @@ export interface AvatarProps extends StyleProps, DOMProps {
   /** Text description of the avatar. */
   alt?: string,
   /** The image URL for the avatar. */
-  src?: string
+  src?: string,
+  /** The stroke color for the avatar. */
+  strokeColor?: 'white' | 'black'
```

Comment:
is this really staticColor?
Though seems like there's actually a whole bunch of options for the strokeColor. Possibly every "layer" color is a choice?
![Screenshot 2024-08-06 at 10 45 21 AM](https://github.com/user-attachments/assets/2bdb9e2a-b78c-4587-96fe-96e99407ba2e)